From f06b35cfa664831c83aad7d78d47d2371173ddf8 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Wed, 19 Jan 2022 13:14:31 -0700
Subject: [PATCH] Use collections.abc rather than collections

---
 pymatgen/db/query_engine.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/pymatgen/db/query_engine.py b/pymatgen/db/query_engine.py
index 452c481..b8ec006 100644
--- a/pymatgen/db/query_engine.py
+++ b/pymatgen/db/query_engine.py
@@ -18,7 +18,8 @@
 import os
 import gridfs
 import zlib
-from collections import OrderedDict, Iterable
+from collections import OrderedDict
+from collections.abc import Iterable
 
 import pymongo
 from pymongo import MongoClient
